binaries/Readme.md

```diff
@@ -111,24 +111,90 @@ Binaries (query engine binary and migration engine binary) are at the core of Ph
+| `rhel`         | CentOS              | 1.0.x, 1.1.y       | (tbd)     |
+
+### Linux distributions and versions
+
+This table shows which Linux distributions are compatible with our pre-built binaries.
+
+| OS                  | Version                  | Build          | Status [?](#status-legend) | OpenSSL [[1]](#openssl) | Comment |
+| :------------------ | :----------------------- | :------------- | :------------------------: | :---------------------: | :------ |
+| Debian              | 8 (Jessie)               | `debian`       | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.0.0`
+|                     | 9 (Stretch)              |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.1`
+|                     | 10 (Buster)              |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `libssl1.1`
+| Ubuntu              | 14.04 (trusty)           | `debian`       | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+|                     | 16.04 (xenial)           |                | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+|                     | 18.04 (bionic)           |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+|                     | 19.04 (disco)            |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+| CentOS              | 6                        | `rhel`         | :heavy_check_mark:         | 1.0.x                   |
+|                     | 7                        |                | :heavy_check_mark:         | 1.0.x                   |
+| Fedora              | 28                       | `rhel`         | :heavy_check_mark:         | 1.1.x                   |
+|                     | 29                       |                | :heavy_check_mark:         | 1.1.x                   |
+|                     | 30                       |                | :heavy_check_mark:         | 1.1.x                   |
+| Linux Mint          | 18                       | `debian`       | (:heavy_check_mark:)       | 1.0.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+|                     | 19                       |                | (:heavy_check_mark:)       | 1.1.x                   | [[2]](#dependencies) install `nodejs` or `openssl`
+| Arch Linux          | 2019.09.01               | `debian`       | :heavy_check_mark:         | 1.1.x                   |
+| Alpine              | *                        | n/a            | :x:                        | ?                       | see [prisma/photonjs#173](https://github.com/prisma/photonjs/issues/173)
+
```

Comment:
I think that we will need to pin down the final names of the binaries before merging this.

Reply:
I can expand the table so that each single will get its own row.